README.md

```diff
@@ -229,11 +235,13 @@ final CardComponentConfiguration cardComponentConfiguration = CardComponentConfi
   amount: AMOUNT,
 );
 ```
-4. Create an `AdvancedCheckout` object and provide two callbacks:<br>
-- `onSubmit`: from your server, make a [`/payments`](https://docs.adyen.com/api-explorer/Checkout/latest/post/payments) request. <br>
-- `onAdditionalDetails`: from your server, make a [`/payments/details`](https://docs.adyen.com/api-explorer/Checkout/71/post/payments/details)
+4. Create an `AdvancedCheckoutPreview` providing two callbacks:
+- `onSubmit`: from your backend, make a [`/payments`](https://docs.adyen.com/api-explorer/Checkout/latest/post/payments) request. The callback returns two parameters:
+  - `data`: payment data that needs to be forwarded.
```

Comment:
How are they forwarding this, with a /payment/details call? 
Needs to be forwarded sounds like they have to do something, but there is no step explaining what to do. I would instead say, `data`: payment data that you need to forward in the [request name]